pyexchange/dydx.py

```diff
@@ -0,0 +1,261 @@
+# This file is part of Maker Keeper Framework.
+#
+# Copyright (C) 2020 MikeHathaway
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+from decimal import Decimal
+
+from pprint import pformat
+
+from dydx.client import Client
+import dydx.constants as consts
+import dydx.util as utils
+
+from pyexchange.api import PyexAPI
+from pymaker import Wad
+from typing import List, Optional
+
+import dateutil.parser
+
+import dydx.constants as consts
+import dydx.util as utils
+
+
+class Order:
+    def __init__(self,
+                 order_id: str,
+                 timestamp: int,
```

Comment:
None of the other pyexchange Order classes seem to have the `timestamp` attribute... Is it necessary that we use it?

I guess it cant hurt us to keep it if it's returned from Dydx as long as there aren't any issues when the keeper posts the order to the `order-history-service`

Reply:
Okex, Okcoin, eToro, Bitso, and Korbit all have a `timestamp` attribute, and they've been able to post successfully to `order-history-service`. Given it's returned from the exchange, and harmless this is a form of future proofing in case we ever want to use order data

Reply:
Timestamp will be helpful for PnL.